src/snapred/backend/recipe/ReductionRecipe.py

```diff
@@ -157,7 +158,9 @@ def execute(self):
             # Cleanup
             outputs.append(sampleClone)
             self._deleteWorkspace(normalizationClone)
-        return outputs
+        data["result"] = True
```

Comment:
is the wrapping necessary? It was an old way we were doing things and I'm not sure I see the value in it anymore.

Reply:
It seems to me that at present, more are wrapped than are passing straight through.  To me it's more important to be consistent.